app/assets/stylesheets/components/sub_nav.scss

```diff
@@ -1,34 +1,49 @@
 .subnav {
-  display: flex;
-  margin-left: 0px;
-  margin-right: 0px;
   border-bottom: 1px solid black;
 
+  @media (min-width: 640px) {
```

Comment:
does 640px mean anything? It seems to be used a lot

Reply:
Its the Gov.uk design system breakpoint. Which interestingly is not the same as bootstraps which  I consider the industry norm